tests/unit/onyxMultiMergeWebStorageTest.js

```diff
@@ -141,7 +141,11 @@ describe('Onyx.mergeCollection() amd WebStorage', () => {
     });
 
     it('setItem() and multiMerge()', () => {
-        expect(localforageMock.storageMap).toEqual({});
+        expect(localforageMock.storageMap).toEqual({
+            test_1: null,
```

Comment:
I'm pretty sure he made this change because now instead of removing all of the keys and values when clearing Storage, the keys remain but all the values are set to null.

Reply:
ahhh, OK. Could you add a code comment about that please?